nex/register_common_authentication_server_protocols.go

```diff
@@ -0,0 +1,28 @@
+package nex
+
+import (
+	"os"
+
+	"github.com/PretendoNetwork/friends-secure/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	ticket_granting "github.com/PretendoNetwork/nex-protocols-common-go/ticket-granting"
+)
+
+func registerCommonAuthenticationServerProtocols() {
+	ticketGrantingProtocol := ticket_granting.NewCommonTicketGrantingProtocol(globals.AuthenticationServer)
```

Comment:
The Friends server uses insecure login, we have to enable it here

```suggestion
	ticketGrantingProtocol := ticket_granting.NewCommonTicketGrantingProtocol(globals.AuthenticationServer)
	
	ticketGrantingProtocol.EnableInsecureLogin()
```

Reply:
Enabled